### PR TITLE
fix: cosmetic issue

### PR DESCRIPTION
**Pull request by maulanasdqn**
**TIKET / ISSUE**
Cosmetic : https://app.clickup.com/t/86ep5wdhv
Upload CV : https://app.clickup.com/t/86ep6v1jv

**STATUS**
- In Progress

**TODO**
- [x] Improve Navbar
- [x] Improve Sidebar
- [x] Improve Global Layout
- [x] Integrate Upload CV
- [ ] Integrate CV Analyzer

**Screenshot**
image
image
image

### CODE REVIEW

File: app/(dashboard)/(routes)/dashboard/page.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+'use client';
```
Comment (dwicaksono) [inline app/(dashboard)/(routes)/dashboard/page.tsx]: ini ga perlu kayaknya mas

File: components/job/create-job/upload-cv.tsx [hunk 2]
```diff
@@ -37,6 +37,9 @@ const UploadCv = () => {
   const handleNext = () => {
     setStep(3);
   };
+
+  console.log(files);
```
Comment (dwicaksono) [inline components/job/create-job/upload-cv.tsx]: nitip ini nanti kalau udah fix ready to merge di hapus yak.

File: components/sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+          ))}
+        </div>
+      </div>
+    </aside>
+  );
+};
+
+export default Sidebar;
+
+// const oldRoutes = [
```
Comment (dwicaksono) [inline components/sidebar.tsx]: ini kalau emang ga dipake lagi di hapus aja gpp

File: app/(dashboard)/(routes)/dashboard/page.tsx (hunk 1 above)
Comment (maulanasdqn) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: okee mas

File: components/job/create-job/upload-cv.tsx (hunk 2 above)
Comment (maulanasdqn) [reply components/job/create-job/upload-cv.tsx]: siapp mas

File: components/sidebar.tsx (hunk 3 above)
Comment (maulanasdqn) [reply components/sidebar.tsx]: okeyss, yang commentar nya ya?

File: app/(dashboard)/(routes)/dashboard/page.tsx (hunk 1 above)
Comment (maulanasdqn) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: image


kalau gak pake use client error gini mas @dwicaksono

Comment (dwicaksono) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: coba rebase dev yang baru

Comment (maulanasdqn) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: sudah mas

image

Comment (dwicaksono) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: pindahin aja hoooksnya mas gpp @maulanasdqn 
Screenshot 2024-04-29 at 11 05 20
Screenshot 2024-04-29 at 11 05 32

Comment (dwicaksono) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: Screenshot 2024-04-29 at 11 07 04

Comment (maulanasdqn) [reply app/(dashboard)/(routes)/dashboard/page.tsx]: updated mas

File: dump.rdb
Comment (dwicaksono) [inline dump.rdb:1]: ini file apa ya ? kalau ga perlu bisa di ignor

File: .nx/cache/18.3.4-nx.linux-x64-gnu.node
Comment (dwicaksono) [inline .nx/cache/18.3.4-nx.linux-x64-gnu.node:1]: ini file kalau ga perlu bisa di ignore

File: dump.rdb
Comment (maulanasdqn) [reply dump.rdb:1]: waduh siap mas sorry sorry sebelumnya

Comment (maulanasdqn) [reply dump.rdb:1]: updated mas

File: .nx/cache/18.3.4-nx.linux-x64-gnu.node
Comment (maulanasdqn) [reply .nx/cache/18.3.4-nx.linux-x64-gnu.node:1]: updated mas

**Pull request merged**